src/style.css

```diff
@@ -1,3 +1,72 @@
+:root {
+  --link-color: #00ACDC;
+  --text-color: #000;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.list {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  text-transform: uppercase;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)